demos/code_local_assistant/README.md

```diff
@@ -24,6 +24,46 @@ Export `codellama/CodeLlama-7b-Instruct-hf`:
 python export_model.py text_generation --source_model codellama/CodeLlama-7b-Instruct-hf --weight-format int4 --config_file_path models/config_all.json --model_repository_path models --target_device GPU --overwrite_models
 ```
 
+> **Note:** Use `--target_device NPU` for Intel NPU or omit this parameter to run on Intel CPU
+
+## Prepare Agentic Model 
+We need specialized model that is able to produce tool calls. For this task we will use Qwen3-8B quantized to int4. We will use automatic pulling of HF models, so export script is not required.
+
+Pull and add `OpenVINO/Qwen3-8B-int4-ov` (Linux):
```

Comment:
Did you check Qwen3-4B? Maybe it would be enough in terms of quality and less demanding in terms of resources.